### PR TITLE
feat: manufacturer account creation backend integration

### PR DESCRIPTION
**Pull request by Joy-Adah**
Backend Integration for Manufacturer Account Creation.

Solves #53

### CODE REVIEW

**Comment by Joy-Adah** [comment]
Gm @zintarh kindly review.

File: frontend/src/components/manufacturer/ManufacturerForm.tsx [hunk 1]
```diff
@@ -7,8 +7,51 @@ import {
...
+    formData.append('manufacturerName', manufacturerName);
+    formData.append('address', address);
+    formData.append('email', email);
+    formData.append('rc', registrationCode);
+    formData.append('phone', phone);
+    if (registrationImage) {
+      formData.append('registrationImage', registrationImage);
+    }
+
+    try {
```
Comment (zintarh) [inline frontend/src/components/manufacturer/ManufacturerForm.tsx:36]: Please can you move this logic to api service

File: frontend/src/components/manufacturer/ManufacturerForm.tsx [hunk 2]
```diff
@@ -7,8 +7,51 @@ import {
...
+        headers: {
+          'Content-Type': 'multipart/form-data',
+        },
+      });
+
+      if (response.status !== 200) {
+        // Todo handle error
+        throw new Error('Failed to create manufacturer');
+      }
+      // Todo handle success
```
Comment (zintarh) [inline frontend/src/components/manufacturer/ManufacturerForm.tsx:48]: Please use the alert hook in the app  for success and error

**Comment by Joy-Adah** [comment]
Hey @zintarh please review

**Review by zintarh — APPROVED** [review]
Lgtm

**Pull request merged**